x/slashing/handler_test.go

```diff
@@ -165,7 +165,7 @@ func TestHandleAbsentValidator(t *testing.T) {
 	// will exist since the validator has been bonded
```

Comment:
Please check this comment also.

Reply:
Could you please elaborate on your comment?

Reply:
I pointed wrong line. Sorry.